docs/html/reference/pip_list.rst

```diff
@@ -32,75 +40,182 @@ Examples
 
 #. List installed packages.
 
-    ::
+    .. tabs::
+
+      .. group-tab:: Unix/macOS
+
+        .. code-block:: console
```

Comment:
This file is not well indented either.  At this point I suggest to either:

* Let your editor/linter autoindent everything if it has such functionality
* Leave it as is for follow-up with auto reST linter

Reply:
I will fix these shortly.